pkg/volume/azure_file/azure_file.go

```diff
@@ -102,11 +102,13 @@ func (plugin *azureFilePlugin) NewMounter(spec *volume.Spec, pod *v1.Pod, _ volu
 }
 
 func (plugin *azureFilePlugin) newMounterInternal(spec *volume.Spec, pod *v1.Pod, util azureUtil, mounter mount.Interface) (volume.Mounter, error) {
-	source, readOnly, err := getVolumeSource(spec)
+	share, secretName, secretNamespace, readOnly, err := getVolumeSource(spec)
 	if err != nil {
 		return nil, err
 	}
-
+	if len(secretNamespace) == 0 {
+		secretNamespace = pod.Namespace
```

Comment:
also, triple check the name/namespace order of parameters and return values... different functions are using different orders :-/